tests/testthat/_snaps/sitrep.md

```diff
@@ -0,0 +1,21 @@
+# pkgdown_sitrep works
+
+    Code
```

Comment:
Since the code is not informative, this is a good place to use `expect_snapshot_output()`